### ISSUE TITLE
Add "CSAF" field to find OASIS CSAF documents

### ISSUE THREAD

**Issue by nightwatchcyber** [securitytxt/security-txt#200]
CSAF is an OASIS draft standard to define a machine readable format for security advisories. It would be something like:
`CSAF: https://psirt.domain.tld/advisories/csaf/`

See: https://github.com/oasis-tcs/csaf/issues/152

**Comment by nightwatchcyber** [comment]
Deferred to the future, once the draft is approved this field can be added to the IANA registry via the standard process

**Comment by santosomar** [comment]
Thank you for considering adding a CSAF reference!

**Comment by tschmidtb51** [comment]
This is now tracked in oasis-tcs/csaf#318.

**Comment by nightwatchcyber** [comment]
Thanks - security.txt is about to become an RFC and this work can being right after that

**Comment by santosomar** [comment]
Thank you 🙏

**Comment by nightwatchcyber** [comment]
The RFC has been published:
https://www.rfc-editor.org/rfc/rfc9116.html

**Comment by santosomar** [comment]
A few examples of security.txt with CSAF references:

- https://www.cisco.com/.well-known/security.txt
- https://www.tibco.com/.well-known/security.txt

**Comment by sthagen** [comment]
This one works:
> * https://www.cisco.com/.well-known/security.txt

That one does not (any more `ERR_TOO_MANY_REDIRECTS` )
> * https://www.tibco.com/.well-known/security.txt

**Comment by santosomar** [comment]
I contacted Tibco and they fixed it.

**Comment by tschmidtb51** [comment]
The value `CSAF` has been added to the registry: https://www.iana.org/assignments/security-txt-fields/security-txt-fields.xhtml
